lib/include/ert/enkf/row_scaling.hpp

```diff
@@ -0,0 +1,33 @@
+/*
+  Copyright (C) 2020  Equinor ASA, Norway.
+
+  The file 'row_scaling.hpp' is part of ERT - Ensemble based Reservoir Tool.
+
+  ERT is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  ERT is distributed in the hope that it will be useful, but WITHOUT ANY
+  WARRANTY; without even the implied warranty of MERCHANTABILITY or
+  FITNESS FOR A PARTICULAR PURPOSE.
+
+  See the GNU General Public License at <http://www.gnu.org/licenses/gpl.html>
+  for more details.
+*/
+
+#ifndef ROW_SCALING_H
+#define ROW_SCALING_H
+
+#ifdef __cplusplus
```

Comment:
Libres does not any longer support to be compiled with C and thus this if becomes unnecessary

Reply:
Good point.